cypress/local-only/tests/accessibility/public/trialSessions/trial-session-details.cy.ts

```diff
@@ -0,0 +1,14 @@
+import { checkA11y } from '../../../../support/generalCommands/checkA11y';
+
+describe('Trial Session Details - Public Accessibility', () => {
+  beforeEach(() => {
+    Cypress.session.clearCurrentSessionData();
+  });
+
+  it('should be free of a11y issues', () => {
+    cy.visit('/trial-session-detail/959c4338-0fac-42eb-b0eb-d53b8d0195cc');
```

Comment:
Does this trial session id come from seed data? Is this allowed for `local-only` tests?

Reply:
It does come from the seed data, and yes. (It is basically just a copy of https://github.com/ustaxcourt/ef-cms/blob/b687cd55eaff5afc841cdbc426d1855058ac197a/cypress/local-only/tests/accessibility/trialSession/judge.cy.ts#L17-L25. But come to think of it, we probably don't need to use it. I can update this in the follow-up PR.